api/app/signals/apps/users/admin.py

```diff
@@ -52,6 +54,21 @@ def ja_nee(value):
         self.message_user(request, 'Created summary CSV file: {}'.format(filename))
         return response
 
+    def get_form(self, request, obj=None, **kwargs):
+        """ Makes sure the category permissions are updated on user form load """
+        CategoryPermissions.create_for_all_categories()
```

Comment:
hmmmmmm... every time we go to the admin this needs to be done? What if we never go to the admin for categories? 

Maybe make it some kind of action on the admin page so that it can be triggered by the administrator?

Reply:
Then it doesn't matter. The only place we are assigning permissions is on the admin page. We're just making sure the permissions exist at the moment we want to use (= assign) them. Before that there is no need for the categories to exist: we will simply not have access to certain categories, which would also be the case would the categories in fact exist, but the permissions aren't assigned yet.

Long story short; we never need the permissions before we've visited the admin page, in which case they're always there.